Documentation~/links.md

```diff
@@ -1,16 +1,8 @@
 # Helpful links
 
-ProBuilder follows many of the same conventions as other 3D modeling software, so if you are an experienced 3D artist, you can read the [ProBuilder toolbar](toolbar.md) section and begin using the ProBuilder tools right away.
+ProBuilder follows many of the same conventions as other 3D modeling software, so if you are an experienced 3D artist, you can read the [Interacting with ProBuilder](overview-ui.md) section and begin using the ProBuilder tools right away.
 
 If you are new to 3D modeling, ProBuilder is a great way to learn. You can watch the tutorial videos on the Unity [Youtube playlist](https://www.youtube.com/playlist?list=PLrJfHfcFkLM8PDioWg_5nmUqQycnVmi58) to get started.
 
-* [Tutorials and documentation](https://docs.unity3d.com/Packages/com.unity.probuilder@4.0/manual/index.html)
 * [Support forum](https://forum.unity.com/forums/world-building.146/)
-* [Twitter](http://www.twitter.com/probuilder3d)
-
-<!--
-
-<span style="color:red">@TODO: Follow up with Gabriel: the **Tutorials and documentation** link opens the documentation only. The **Youtube playlist** link opens Gabriel's old Procore video channel, which hasn't had new videos in years. It would be great to create a new ProBuilder YouTube channel where we could promote the most recent videos.</span>
-
--->
-
+* [ProBuilder 3D on X](https://x.com/probuilder3d)
```

Comment:
Looks good, Gabe said there's also an active Discord channel that might be worth adding: https://discord.com/channels/489222168727519232/892763390563467326

Reply:
```suggestion
* [ProBuilder 3D on X](https://x.com/probuilder3d)
* [The ProBuilder Discord channel](https://discord.com/channels/489222168727519232/892763390563467326)
```